README.md

```diff
@@ -1 +1,375 @@
+
+# 1 Introduction
+
+## 1.1 Motivation
+
+[IPLD] is a format for describing Merkle DAGs with semantics similar to CBOR plus hash linking to other blocks of IPLD. It is an efficient machine encoding, but can be cumbersome to work with in the broader tools ecosystem and developer ergonomics.
+
+### 1.1.1 Containers
+
+IPLD assumes that links will be referenced against a blockstore. The resulting disconnected structures — while efficient for machines — can be cumbersome to work with, debug, inspect, and transmit in non-IPLD aware applications.
+
+The ecosystem of tools for non-IPLD specific tools (JSON tools, CBOR tools, etc) is much wider than those designed for IPLD in particular. To make use of them today, a developer either needs to write custom code, or find an ad hoc method for reifying an IPLD DAG into some format without links such as JSON. Packaging data for 
+
+### 1.1.2 Codecs
+
+By referencing a specific [CID] in a link, the encoding and hash algorithm of that block are forced to be decided ahead of time. While common codecs are supported by most implementations, not all applications will support all possible encodings.
+
+The identity hash has been used for similar use cases, but this has several significant drawbacks including:
+
+- A codec and hash algorithm MUST be decided in advance
+- The resulting CID will not directly work with non-IPLD tools (at minimum there's a header)
+- Depending on the parameters other than hash, the data may be illegible without further processing
+- The identity CID is frozen by the Merkle root of the encoding structure
+
+## 1.2 Inline Links
+
+IPLD Inline Links solve for the above by adding a standard way to inline data in an IPLD document without altering its hash. The ability to inherit the CID configuration (hash, algorithm, base encoding, etc) from the surrounding context is included, but is OPTIONAL.
+
+Inline links do not modify the IPLD data model itself, but rather extend the format in a way that can be round-tripped to and from standard IPLD as desired.
+
+# 2 Format
+
+The basic structure of an inline link is similar to how reference links are represented in the most common IPLD codecs: signalling a link by wrapping the payload in a map with the single `"/"` key. By convention, this signals an IPLD-specific extension to a base codec.
+
+## 2.1 Inline Wrapper
+
+An inline link MUST be signalled by wrapping in a map with a `"/"` key.
+
+``` ipldsch
+type InlineWrapper struct {
+  inline InlineLink (rename "/")
+}
+```
+
+## 2.2 Inlined DAG Payload
+
+The inlined DAG payload MUST contain the inlined DAG. The CID MAY be present, omitted, or set to `null`.
+
+``` ipldsch
+type InlineLink struct {
+  cid nullable Link (implicit Null)
+  dag          Any
```

Comment:
`data` makes sense 👍 

> The “.” also seems somewhat in place.

Yeah that was actually the original draft — I had advoctaed for the `{"/": ..., ".": ...}` version in the FAQ back in Brussels because it's so short.

I see that there's some comments about the exact layout, so I'll put a pin in this discussion until we get there